geomesa-hbase/geomesa-hbase-tools/bin/install-hadoop-hbase.sh

```diff
@@ -162,17 +182,29 @@ if [[ -z "${install_dir}" ]]; then
   echo "${usage}"
   exit 1
 else
-  read -r -p "Install hadoop dependencies to ${install_dir}?${NL}Confirm? [Y/n]" confirm
+  read -r -p "Install Hadoop, HBase, and Zookeeper dependencies to ${install_dir}?${NL}Confirm? [Y/n]" confirm
   confirm=${confirm,,} # Lowercasing
   if [[ $confirm =~ ^(yes|y) || $confirm == "" ]]; then
     # Setup download URLs
     declare -a urls=(
+      "${base_url}org/apache/hbase/hbase-common/${hbase_version}/hbase-common-${hbase_version}.jar"
+      "${base_url}org/apache/hbase/hbase-protocol/${hbase_version}/hbase-protocol-${hbase_version}.jar"
+      "${base_url}org/apache/hbase/hbase-client/${hbase_version}/hbase-client-${hbase_version}.jar"
```

Comment:
I think we want to keep the hbase install separate too - in geoserver, we have to shade in hbase, so we only need the hadoop jars but not hbase